### PR TITLE
fix: play cards ui 

### PR DESCRIPTION
**Pull request by Akshaygore1**
> First thing, PLEASE READ THIS: [ReactPlay Code Review Checklist](https://github.com/reactplay/react-play/wiki/ReactPlay-Code-Review-Checklist)

# Description
https://github.com/reactplay/react-play/issues/1447
After taking the lead on the issue, I created a new Card component called playCard and passed props to it, resulting in a change in the UI.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce.

You Can go to /play route and see the changes 

# Checklist:

- [x
ReactPlay - Plays · 02 51 · 02-17
] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

# Screenshots or example output

### CODE REVIEW

File: src/plays/quiz-app/QuizApp.tsx
Comment (priyankarpal) [inline src/plays/quiz-app/QuizApp.tsx:1]: there are a few formatting issues please fix those issues

**Review by priyankarpal — CHANGES_REQUESTED** [review]

File: src/plays/quiz-app/QuizApp.tsx
Comment (Akshaygore1) [reply src/plays/quiz-app/QuizApp.tsx:1]: Fixed

File: src/common/playlists/PlayCard.jsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { BsPlayCircleFill } from 'react-icons/bs';
+import { BiLogoTypescript } from 'react-icons/bi';
+import { BiLogoJavascript } from 'react-icons/bi';
```
Comment (priyankarpal) [inline src/common/playlists/PlayCard.jsx:5]:
```suggestion
import { BiLogoJavascript, BiLogoTypescript } from 'react-icons/bi';
```

File: src/common/playlists/PlayCard.jsx [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+            <div className="author-name">{play?.user.displayName}</div>
+          </div>
+        )}
+        <div className="play-actions mt-4">
+          <div className="like-container">
+            <Like likeObj={likeObject()} onLikeClick={null} />
+            {play.language === 'ts' ? (
+              <BiLogoTypescript size={25} />
+            ) : (
+              <BiLogoJavascript size={25} />
```
Comment (priyankarpal) [inline src/common/playlists/PlayCard.jsx:49]: if the language is `ts` then the hover color should be `#007acc` otherwise `#F0DB4F`

**Review by priyankarpal — CHANGES_REQUESTED** [review]

**Comment by Akshaygore1** [comment]
I have updated the request as per your instructions regarding the changes.

**Comment by priyankarpal** [comment]
> I have updated the request as per your instructions regarding the changes. @priyankarpal

there is a little bit issue just need to `left-align` the project name 
image

**Review by priyankarpal — CHANGES_REQUESTED** [review]
We can use Tailwind CSS instead of writing vanilla CSS

File: src/common/playlists/PlayCard.jsx [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+      <div className="play-card-container">
+        <div className="play-thumb-container">
+          <Shimmer>
+            <img alt="" className="play-card-thumb-img" src={cover} />
+          </Shimmer>
+          <BsPlayCircleFill className="play-icon" color="white" size={80} />
+        </div>
+
+        {/* <div className="border" /> */}
+        <div className="card-header">{play.name}</div>
```
Comment (priyankarpal) [inline src/common/playlists/PlayCard.jsx:22]: need to make the `play` name to `left-align` it's a dynamic name

**Review by priyankarpal — CHANGES_REQUESTED** [review]

File: src/common/playlists/PlayCard.jsx (hunk 3 above)
Comment (Akshaygore1) [reply src/common/playlists/PlayCard.jsx:22]: it will not look good as in design we often seen mixture of centre and side aligned

**Comment by Akshaygore1** [comment]
> We can use Tailwind CSS instead of writing vanilla CSS

we can

**Comment by Akshaygore1** [comment]
q

> > We can use Tailwind CSS instead of writing vanilla CSS
> 
> > > We could, but I didn't use it because the properties of the icon are not being overridden.

**Comment by priyankarpal** [comment]
> > We can use Tailwind CSS instead of writing vanilla CSS
> 
> We could, but I didn't use it because the properties of the icon are not being overridden.

ok

File: src/common/playlists/PlayCard.jsx (hunk 3 above)
Comment (priyankarpal) [reply src/common/playlists/PlayCard.jsx:22]: try once & send us the UI design here `before` & `after`

Comment (Akshaygore1) [reply src/common/playlists/PlayCard.jsx:22]: Screenshot 2024-02-21 at 21 11 46
Screenshot 2024-02-21 at 21 12 10

Comment (Akshaygore1) [reply src/common/playlists/PlayCard.jsx:22]: this is before and after

Comment (Akshaygore1) [reply src/common/playlists/PlayCard.jsx:22]: done

Comment (Akshaygore1) [reply src/common/playlists/PlayCard.jsx:22]: I think we are Good to Go

Comment (Akshaygore1) [reply src/common/playlists/PlayCard.jsx:22]: @priyankarpal

**Review by priyankarpal — APPROVED** [review]
Looks Good 🎉 

@Akshaygore1 I will suggest to name branches like this format `fix-issue-issuenumber`

**Comment by priyankarpal** [comment]
@all-contributors please add @Akshaygore1 as a contributor for code

**Pull request merged**